README.rst

```diff
@@ -1402,9 +1402,19 @@ Choose from the list of available rules:
     ``'newest'``
   - ``use_class_const`` (``bool``): use ::class notation; defaults to ``true``
 
-* **php_unit_ordered_covers** [@PhpCsFixer]
+* **php_unit_ordered_annotations** [@PhpCsFixer]
 
-  Order ``@covers`` annotation of PHPUnit tests.
+  Order annotations of PHPUnit tests.
+
+  Configuration options:
+
+  - ``annotations`` (a subset of ``['covers', 'uses']``): list of annotations to
```

Comment:
Perhaps other annotations also make sense (for example, `@dataProvider`)?

Reply:
maybe add these: `@group`, `@author`, `@dataProvider` ?

Reply:
Crazy thing is that I heard from @epdenouden that some people write tests where the order in which data providers are configured matter.

😆

Reply:
@SpacePossum 

Not sure about `@author` - would this make sense outside of the scope of a `phpunit/phpunit` test case?

Reply:
well that is .. ehmm .... interesting XD    anyway... they can omit the option than : P

Reply:
Im definitely fine with leaving `author` out, I was more typing out loud than pushing specs :)

Reply:
@SpacePossum 

Had not been aware that `@author` is a thing as an annotation for `phpunit/phpunit`!

<img width="1104" alt="Screen Shot 2019-10-11 at 21 29 56" src="https://user-images.githubusercontent.com/605483/66679392-5f3e8000-ec6e-11e9-986c-f22a33f9472c.png">

Reply:
Not a fan of using `@depends`, but does anyone know whether one can specify multiple `@depends`?

Reply:
I never new it was an alias, TIL :D

Reply:
https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.examples.MultipleDependencies.php

some a sample of mulitple depends (not a big fan either btw)

Reply:
@SpacePossum 

Added~